### DISCUSSION TITLE
Comparison to other wallet connect kits?

### DISCUSSION THREAD

**Discussion by songkeys** [family/connectkit#12] [Q&A] [answered]
Nice work. I want to try but I have used another react wallet connect kit (with more stars and usage coverage). It'd be better if we put a comparison part on the doc to illustrate the differences between these libraries. Specifically, why "connectkit" over others? That'd be much helpful for our devs.

**Comment by lochie** [comment, marked as answer]
Great suggestion! We actually already have a backlog item to implement a comparison table in our docs, so this will most likely be added in the near future.
Thanks for the feedback!

**Reply by songkeys** [reply]
That'd be great. Will keep watching this repo!
